setup.py

```diff
@@ -14,7 +14,7 @@
     license='MIT',
     description='SendGrid Backend for Django',
     long_description=open('./README.rst').read(),
-    install_requires=["sendgrid >= 3.5, < 4"],
+    install_requires=["sendgrid == 5.2.0"],
```

Comment:
Since the library follows [semantic versioning](http://semver.org/), there shouldn't be any breaking changes in any 5.x.x version. I think `sendgrid >= 5 , < 6` would be better here.

Reply:
I'll make the change - I would advocate for freezing dependencies though. Imo better to assume the maintainers will make an error and defend against it.